app/views/pages/help.html.slim

```diff
@@ -2,80 +2,193 @@
+  li = t('pages.help.creating_account.answer_4_part_2')
 
-h3 = t('headings.help.signing_in.forgot_password')
+h3 = t('pages.help.creating_account.question_5')
 
-h3 = t('headings.help.signing_in.miss_passcode')
+p = t('pages.help.creating_account.answer_5')
 
-h3 = t('headings.help.signing_in.lose_phone')
+ul
 
-h3 = t('headings.help.signing_in.log_out')
+  li = t('pages.help.creating_account.answer_5_part_1_html', update_phone_number_link: link_to(t('pages.help.creating_account.update_phone_number_link'), manage_phone_path))
 
-h3 = t('headings.help.signing_in.forgot_codes')
+  li = t('pages.help.creating_account.answer_5_part_2_html', resend_confirmation_link: link_to(t('pages.help.creating_account.resend_confirmation_link'), '#'))
 
-h2 = t('headings.help.changing_settings.title')
+h3 = t('pages.help.creating_account.question_6', app: APP_NAME)
 
-h3 = t('headings.help.changing_settings.change_password')
+p = t('pages.help.creating_account.answer_6', app: APP_NAME)
 
-h3 = t('headings.help.changing_settings.change_email')
+h3 = t('pages.help.creating_account.question_7', app: APP_NAME)
 
-h3 = t('headings.help.changing_settings.edit_profile')
+p = t('pages.help.creating_account.answer_7_part_1', app: APP_NAME)
 
-h2 = t('headings.help.privacy_security.title')
+p = t('pages.help.creating_account.answer_7_part_2', app: APP_NAME)
 
-h3 = t('headings.help.privacy_security.protect_data')
+h3 = t('pages.help.creating_account.question_8')
 
-h3 = t('headings.help.privacy_security.2fa')
+p = t('pages.help.creating_account.answer_8_part_1', app: APP_NAME)
 
-h3 = t('headings.help.privacy_security.third_party')
+p = t('pages.help.creating_account.answer_8_part_2')
 
-h3 = t('headings.help.privacy_security.remove_saved')
+p = t('pages.help.creating_account.answer_8_part_3', app: APP_NAME)
 
-h2 = t('headings.help.questions.title')
+h3 = t('pages.help.creating_account.question_9')
 
-p = t('pages.help.questions.paragraph_1')
+p = t('pages.help.creating_account.answer_9')
+
+h2 = t('pages.help.signing_in.title')
+
+h3 = t('pages.help.signing_in.question_1')
+
+p = t('pages.help.signing_in.answer_1', app: APP_NAME)
+
+h3 = t('pages.help.signing_in.question_2')
+
+p = t('pages.help.signing_in.answer_2_part_1')
+
+p = t('pages.help.signing_in.answer_2_part_2', app: APP_NAME)
+
+h3 = t('pages.help.signing_in.question_3')
+
+p = t('pages.help.signing_in.answer_3')
+
+h3 = t('pages.help.signing_in.question_4')
+
+p = t('pages.help.signing_in.answer_4_part_1')
+
+p = t('pages.help.signing_in.answer_4_part_2')
+
+h3 = t('pages.help.signing_in.question_5')
+
+p = t('pages.help.signing_in.answer_5_part_1')
+
+p = t('pages.help.signing_in.answer_5_part_2')
+
+p = t('pages.help.signing_in.answer_5_part_3')
+
+h3 = t('pages.help.signing_in.question_6')
+
+p = t('pages.help.signing_in.answer_6_part_1')
```

Comment:
In this case, it would be more of an organization thing than something to increase readability (would read exactly the same to end user)